FirebaseAuth/Sources/Swift/MultiFactor/Phone/PhoneMultiFactorInfo.swift

```diff
@@ -55,10 +55,10 @@ import Foundation
     override public class var supportsSecureCoding: Bool { return secureCodingWorkaround }
 
     public required init?(coder: NSCoder) {
-      guard let phoneNumber = coder.decodeObject(forKey: kPhoneNumberCodingKey) as? String else {
+      guard let phoneNumber = coder.decodeObject(forKey: kPhoneNumberCodingKey) as? NSString else {
```

Comment:
Is `NSString` available in Swift?

Reply:
Yes. It addresses a runtime warning with NSCoder. It still gets converted to String when assigned to phoneNumber.

Reply:
Good to know, thanks.